hs_core/search_indexes.py

```diff
@@ -356,3 +378,166 @@ def prepare_editors_count(self, obj):
             return obj.raccess.edit_users.all().count()
         else:
             return 0
+
+    def prepare_geometry_type(self, obj):
+        if hasattr(obj, 'metadata'):
+            if isinstance(obj.metadata, GeographicFeatureMetaData):
+                return obj.metadata.geometryinformation.all().first().geometryType
```

Comment:
This can raise exception. Better to check that the object exists before accessing its attribute ```geometryType```